airbyte-integrations/connectors/source-instagram/unit_tests/test_common.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
+import pytest
```

Comment:
I don't see any usage for that import. What is it there for?

Reply:
Fixed